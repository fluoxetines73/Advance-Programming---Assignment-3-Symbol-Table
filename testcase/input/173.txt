INSERT c number
ASSIGN c 987654
RPRINT