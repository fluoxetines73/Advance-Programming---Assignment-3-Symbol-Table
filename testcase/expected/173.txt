success
success
c//0